src/NavBar.js

```diff
@@ -1,7 +1,32 @@
+import { NavLink } from "react-router-dom";
+
 function NavBar() {
+  const navStyles = {
+
+  };
+
+  const activeStyles = {
+    background: "#e3d9c3",
+    fontWeight: "bold"
+  };
+
+  const linkStyles = {
+    textDecoration: "none",
+    color: "white",
+    background: "#384b96",
+    display: "inline-block",
+    width: "50%",
+    padding: "20px 0",
+    fontSize: "30px",
+    letterSpacing: "10px",
+    textAlign: "center",
+    lineHeight: "1"
+  };
 
   return (
-    <div>NAVBAR TEST</div>
+    <div className="navBar" style={navStyles}>
```

Comment:
normally having something like `className="navBar"` would achieve the styling, meaning there's something in place to grab styling for a className "navBar". However, we're also feed it the navStyles object. 

It achieves the goal but optimally, there's a styling library or something along those lines (or straight up css rules) that takes care of loading up the right set of styles for the class. We prefer this b/c we usually only want to reserve the `style` prop to _override_ some style rules, making it very clear "this is a bit different from the norm"

Not something I'd have you take care of right away tbh, but we'll do a CSS crash course type of thing in the near future